Docs/Configuration.tex

```diff
@@ -3154,6 +3154,13 @@ \subsection{Properties}\label{miscprops}
   \textbf{Description}: Apply the security configuration described in the
   \hyperref[miscsecurityprops]{Security Properties} section below.
 
+\item
+  \texttt{Serial}\\
+  \textbf{Type}: \texttt{plist\ dict}\\
+  \textbf{Description}: Update the PCD values listed in
+  \href{https://github.com/acidanthera/audk/blob/master/MdeModulePkg/Library/BaseSerialPortLib16550/BaseSerialPortLib16550.inf}{BaseSerialPortLib16550.inf}.
+  Refer to \href{https://github.com/acidanthera/audk/blob/master/MdeModulePkg/MdeModulePkg.dec}{MdeModulePkg.dec} for the explanations of each key.
```

Comment:
I'm not sure we should mess much with mentioning PCDs or MdeModulePkg.dec. This feels like implementation details to the OC serial config. Cc @vit9696?

Reply:
Better to describe the meaning than just ref the pcd. Still I think referencing the pcd is helpful.

Reply:
They are just in MdeModulePkg.dec, so I think the reference to it is enough. Otherwise it will be more about copy-pasting, no?

Reply:
Still better for the end user to at least give a description in my opinion.

Reply:
OK, I added brief descriptions, by mostly copy-pasting at https://github.com/acidanthera/OpenCorePkg/commit/ceb042e6a3f729ce55bd437426ef8fad4d45d953. :p